cmd/queue/main.go

```diff
@@ -276,6 +276,8 @@ func probeQueueHealthPath(port int, timeoutSeconds int) error {
 }
 
 func main() {
+	reporterReportingPeriod := flag.Int64("reportingperiod", 1, "specify only integers greater than 1 if ReporterReportingPeriod is not 1 second")
+	fmt.Println(strconv.FormatInt(*reporterReportingPeriod, 10))
```

Comment:
As dicussed in Slack, this is not needed for now. Let's add a constant in this file being the reportingperiod.

Reply:
Changed